pallets/attestation/src/lib.rs

```diff
@@ -465,6 +471,19 @@ pub mod pallet {
 				ExternalAttestations::<T>::remove(authorization_id, claim_hash);
 			}
 		}
+
+		#[cfg(any(feature = "try-runtime", test))]
+		pub fn do_try_state() -> Result<(), &'static str> {
```

Comment:
These do not need to be public, so making them all crate-visible is the right thing.

```suggestion
		pub(crate) fn do_try_state() -> Result<(), &'static str> {
```